packages/format-library/package.json

```diff
@@ -0,0 +1,36 @@
+{
+	"name": "@wordpress/format-library",
+	"version": "2.1.0",
+	"description": "Format library for the WordPress editor.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"formats"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/format-library/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"main": "build/index.js",
+	"module": "build-module/index.js",
+	"react-native": "src/index",
+	"dependencies": {
+		"@babel/runtime": "^7.0.0",
+		"@wordpress/components": "file:../components",
+		"@wordpress/dom": "file:../dom",
+		"@wordpress/editor": "file:../editor",
```

Comment:
Oh, so `@wordpress/editor` is listed here. Again, both packages depend on each other. It needs to be fixed.

Reply:
I don't think this is the case?